BUILD.bazel

```diff
@@ -30,10 +30,19 @@ gazelle(
 filegroup(
     name = "node_modules",
     # Only include files needed for type-checking and runtime
-    srcs = glob([
-        "node_modules/**/*.js",
-        "node_modules/**/*.d.ts",
-        "node_modules/**/*.json"
-    ]),
+    srcs = glob(["/".join(["node_modules", pkg, "**", ext]) for pkg in [
+        "jasmine",
+        "typescript",
+        "@types",
+        "tsickle",
+        "tsutils",
+        "source-map-support",
+        "protobufjs",
+        "bytebuffer",
+    ] for ext in [
+        "*.js",
+        "*.json",
+        "*.d.ts",
+    ]]),
```

Comment:
Is this related to this change?

Reply:
I was getting errors on Mac with exceeding max open file descriptors.
I'll make a separate change for this, or even better, figure out why bazel/ibazel have so many open FDs...